### ISSUE TITLE
Dependency `pint` moved location of `units.py` with v0.20

### ISSUE THREAD

**Issue by kyleredilla** [Ouranosinc/xclim#1216]
### Generic Issue

* xclim version: 0.38.0
* Python version: 3.9.13
* Operating System: CentOS 7.9.2009 

### Description

Tried to import `xclim.indices` and got an import error for dependency `pint.unit`. 

It looks like `pint.unit` has moved to `pint/facets/plain/unit.py` with version 0.20. I'm not actually sure if this is an issue with `xclim` or `conda` but figured I'd share. 

[`pint` 0.19.2 <- 0.20 comparison](https://github.com/hgrecco/pint/compare/0.19.2...0.20#diff-4c0f141ab6194c438ee5f28e7308eef5a1870b4e5bf0e239730cf5f657f2ddff)

Also thank you, this package is amazing, hope I can contribute one day. 

### What I Did

Built new conda environment with:

```
conda create -n new_env python=3.9.13 xarray netcdf4 xclim <other packages>
```

Then imported `xclim.indices` in a Jupyter notebook:

```
import xclim.indices as xci
```

### What I Received

I got an import error, didn't think to save the whole thing but it was this at the root:

```
ModuleNotFoundError: No module named 'pint.unit'


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aulemahal** [comment]
Hi @kyleredilla !

Thanks for the issue. Indeed, there was a refactor in `pint` 0.20 and it broke xclim... Good news is that it was fixed in #1212!

We had a plan to publish a new version of xclim (0.39) tomorrow, but I fear the Pull Request queue is a bit loaded and it might have to wait a few more days... Downgrading pint should be a simple temporary fix, if that is possible for you!

**Comment by kyleredilla** [comment]
Ahh I figured this may have been already fixed. Yep already working with the downgrade. Thank you!
